transformer_engine/jax/fp8.py

```diff
@@ -214,13 +214,6 @@ def finalize() -> None:
         FP8 helper finalize
         """
         FP8Helper.INITIALIZED = False
-        FP8Helper.MARGIN = 0.0
```

Comment:
I still suggest to reset these variables after context-manager, since this is the expection from end users to context-manager (Things only change inside the context and turn to original setup after exiting.)

Reply:
Removed.